js/Data/xhrNew.json

```diff
@@ -0,0 +1,1082 @@
+{
+    "textareas": {
+        "data": [
+            {
+                "label": "Should the candidate be hired?",
+                "placeholder": "The type of project that is suitable for the candidate Is guidance required for the candidate",
+                "name": "beHired"
+            },
+            {
+                "label": "General impression",
+                "placeholder": "*required",
+                "name": "generalImpression"
+            },
+            {
+                "label": "Workflow, leadership and Soft skills",
+                "placeholder": "Describe the environment in which the candidate works. Describe any guidance or management experience.",
+                "name": "workflow"
+            }
+        ]
+    },
+    "technical": {
+        "title": "Technical level",
+        "headers": [
+            "Trainee",
+            "Junior",
+            "Middle",
+            "Senior"
+        ],
+        "columnData": [
+            {
+                "inputLevels": [
+                    "trainee"
+                ]
+            },
+            {
+                "inputLevels": [
+                    "junior1",
+                    "junior2",
+                    "junior3"
+                ]
+            },
+            {
+                "inputLevels": [
+                    "middle1",
+                    "middle2",
+                    "middle3"
+                ]
+            },
+            {
+                "inputLevels": [
+                    "senior1",
+                    "senior2",
+                    "senior3"
+                ]
+            }
+        ]
+    },
+    "fieldsets": {
+        "data": [
+            {
+                "groupName": "OOP, Design Patterns",
+                "items": [
+                    {
+                        "label": "Classes",
+                        "name": "classes",
+                        "val_lab": [
```

Comment:
In some situations, having specific values per each dropdown might be the best / only option, but I'd argue that in our particular case, we can have just a single property (say `"options"` / `"gradingOptions"` / `"feedbackRankings"` / `"rankings"` etc) that we could reuse for each of these `<select>` boxes, since all of them share the exact same values, unless I'm missing something. Just for consideration, for a bit of cleanup.